Sources/Identity.swift

```diff
@@ -42,15 +43,58 @@ import Foundation
     }
 
     public func readyForEvent(_ event: Event) -> Bool {
+        if event.urlVariables, getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event)?.value == nil { return false}
```

Comment:
we should rather check that the config shared state is set, e.g. getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event)?.status != .set { return false }